### DISCUSSION TITLE
Syntax error in docs

### DISCUSSION THREAD

**Discussion by Matthewenderle** [supabase/supabase#12189] [General]
I'm brand new to Supabase and while looking for how to order results I stumbled upon a syntax error.

https://supabase.com/docs/reference/dart/order

Docs state the second parameter to `.order` is simply `ascending: false` instead of being in brackets.

```js
final data = await supabase
  .from('cities')
  .select('name, country_id')
  .order('id',  ascending: false );
  ```
  
  It should read:
  
```js
final data = await supabase
  .from('cities')
  .select('name, country_id')
  .order('id',  { ascending: false } );
  ```

**Comment by j4w8n** [comment]
You can look through the issues and pull requests. Someone may already have something for this. If not, feel free to open your own pull request with the fix if you're comfortable doing that.

**Reply by Matthewenderle** [reply]
I tried looking for the repo that contained the docs but I couldn't find it. Am I just overlooking something?

**Reply by Matthewenderle** [reply]
I guess since my chance to do a pull request was sniped, I looked at @gefgu and now I got my answer.

**Reply by j4w8n** [reply]
@Matthewenderle, I'm sorry that happened.

If you're interested in helping solve other issues, there are def opportunities. I find it fun and rewarding.

Depending on your knowledge, you can go to the main supabase github page and find plenty of repos to look through. I'm more familiar with the gotrue-js authentication stuff.

**Comment by gefgu** [comment]
@j4w8n Good reminder! I gave a look through the issues and pull requests and because there were none regarding this issue, I made my own - I hope it solves the issue :)
